docs/carbon-as-spark-datasource-guide.md

```diff
@@ -96,6 +96,30 @@ df.write.format("carbon").save("/user/person_table")
 val dfread = spark.read.format("carbon").load("/user/person_table")
 dfread.show()
 ```
+## Supported OPTIONS
+
+| Property                          | Default Value                             | Description                                                                                                                                                                                                                                                                                                                                                                       |
+|-----------------------------------|-------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| table_blocksize                   | 1024                                      | Size of blocks to write onto hdfs. For more details, see [Table Block Size Configuration](./ddl-of-carbondata.md#table-block-size-configuration).                                                                                                                                                                                                                                 |
```

Comment:
i think, since already there is one table above in this md file with same contents, you can add a link for those properties and add only the remaining properties in this table

Reply:
Done